brozzler/worker.py

```diff
@@ -260,6 +259,45 @@ def thumb_jpeg(self, full_jpeg):
         img.save(out, "jpeg", quality=95)
         return out.getbuffer()
 
+    def should_ytdlp(self, logger, site, page, page_status, skip_av_seeds):
+        # called only after we've passed needs_browsing() check
+
+        # We definitely shouldn't ytdlp if the optional extra is missing
+        try:
+            import yt_dlp
```

Comment:
a better place to do this import optionally is likely at the beginning of the worker, around where we set up prometheus metrics, just after `enable_youtube_dl` is set

Reply:
Sounds good, maybe I'll put it in the constructor and have it forcibly turn `self._skip_youtube_dl` on if the import is missing.

Reply:
How's 1f89a917734c54e3633657a83e3dc6bab26d4865 look?